src/Workspaces/Core/Portable/Editing/ImportAdderService.cs

```diff
@@ -142,17 +145,19 @@ private SyntaxNode GenerateNamespaceImportDeclaration(INamespaceSymbol namespace
         /// </summary>
         /// <param name="root">ref as we add simplifier annotations to nodes with explicit namespaces</param>
         /// <returns></returns>
-        private (ImmutableArray<SyntaxNode> imports, IEnumerable<INamespaceSymbol> namespaceSymbols, SyntaxNode? context) GetImportDirectivesFromSyntaxes(
+        private async ValueTask<(ImmutableArray<SyntaxNode> imports, IEnumerable<INamespaceSymbol> namespaceSymbols, SyntaxNode? context, SemanticModel model, SyntaxNode? newRoot)> GetImportDirectivesFromSyntaxesAsync(
+            Document document,
```

Comment:
indentation off.

Reply:
This is what both the indenter and formatter gave. The indentation of the other lines is off but they are not part of this pull request.

Reply:
please fix the other lines.  do not check in purposefully inconsistent stuff like this.  it's just bad.